__tests__/app.test.js

```diff
@@ -121,4 +121,50 @@ describe('GET /api/articles', () => {
                 expect(articles).toBeSortedBy('created_at', { descending: true });
             });
     });
+});
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: responds with an array of comments with the correct properties', () => {
+        return request(app)
+            .get('/api/articles/1/comments') 
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.comments).toBeInstanceOf(Array);
+                body.comments.forEach((comment) => {
+                    expect(comment).toHaveProperty('comment_id');
+                    expect(comment).toHaveProperty('votes');
+                    expect(comment).toHaveProperty('created_at');
+                    expect(comment).toHaveProperty('author');
+                    expect(comment).toHaveProperty('body');
+                    expect(comment).toHaveProperty('article_id'); 
```

Comment:
also test for the type of the property e.g.

.toHaveProperty('votes', expect.any(Number))